### PR TITLE
get filtered events

### PR DESCRIPTION
**Pull request by haroun-b**

### CODE REVIEW

File: models/Event.model.js [hunk 1]
```diff
@@ -9,12 +9,21 @@ const eventSchema = new Schema(
...
+      type: {
+        type: String,
+        enum: ['Point'],
+        default: 'Point'
       },
+      coordinates: {
+        type: [Number],
+        required: true
+      }
     },
```
Comment (ArthurVEROT) [inline models/Event.model.js:27]: I don't understand the modification on location: 
No more lat long ?
What's type and enum 'Point' ?

File: models/Event.model.js [hunk 2]
```diff
@@ -23,8 +32,15 @@ const eventSchema = new Schema(
...
+      minimum: {
+        type: Number,
+        min: 1,
+        default: 1
+      },
+      maximum: {
+        type: Number,
+        min: 1,
+      }
     },
```
Comment (ArthurVEROT) [inline models/Event.model.js:44]: Good stuff !

File: routers/events.router.js [hunk 3]
```diff
@@ -2,13 +2,135 @@ const router = require("express").Router();
...
+    )
+      .populate({
+        path: `creator`,
+        select: { password: 0, email: 0, __v: 0 }
+      });
+
+    // an array of promises when resolved becomes a nested array where every sub-array has all the approved attendees for the event with the same index in filteredEvents
+    const allEventsAttendeesPromises = []
+
+    filteredEvents.map(evnt => {
```
Comment (ArthurVEROT) [inline routers/events.router.js:122]: isn't it better to use a forEach if you don't get the result of the map ?

File: models/Event.model.js (hunk 1 above)
Comment (haroun-b) [reply models/Event.model.js:27]: This is how GeoJSON does it.
https://thecodebarbarian.com/80-20-guide-to-mongodb-geospatial-queries

**Review by ArthurVEROT — APPROVED** [review]

**Pull request merged**